tests/test_sessions.py

```diff
@@ -0,0 +1,92 @@
+
+import bayeslite
+import bdbcontrib
+from bdbcontrib import sessions
+
+# TODO MAKE INTO AN ACTUAL TEST
```

Comment:
Can you say more about what would make this into an actual test that isn't here now?